solver/exec.go

```diff
@@ -78,7 +78,7 @@ func (e *execOp) Run(ctx context.Context, inputs []Reference) ([]Reference, erro
 			mountable = ref
 		}
 		if m.Output != pb.SkipOutput {
-			if m.Readonly && ref != nil {
+			if m.Readonly && ref != nil && m.Dest == pb.RootMount { // exclude read-only rootfs
```

Comment:
shouldn't this be `m.Dest != pb.RootMount`

Reply:
ping @AkihiroSuda

Reply:
I'll update this night, sorry 😅